packages/terra-collapsible-menu-view/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Added aria-haspopup attribute to menu buttons.
```

Comment:
nit: Can we also add the "Why" here?

Reply:
Updated here: https://github.com/cerner/terra-framework/pull/1760/commits/8f78d1fa23dc6f412e23421b487e8dec0b49a9bb